docs/code_overview/models/liveupdate.rst

```diff
@@ -0,0 +1,5 @@
+LiveUpdate
```

Comment:
Two commits (LiveUpdate + docs) are squashed into one, and changelog entry is added.